lang/py/src/avro/schema.py

```diff
@@ -857,6 +857,8 @@ def make_avsc_object(json_data, names=None):
         if logical_type == 'decimal':
           precision = json_data.get('precision')
           scale = 0 if json_data.get('scale') is None else json_data.get('scale')
+          other_props['name'] = name
+          other_props['namespace'] = namespace
```

Comment:
The name and namespace are the required fields of fixed schema. They're not the other_props.